react/utils/index.js

```diff
@@ -21,10 +21,14 @@ export function setStyle(target, styles) {
   })
 }
 
+function _getZeroOrString(num = 0) {
```

Comment:
I don't really like this `_` to mean, I suppose, that this function is private. It's unnecessary, especially considering that this function is not being exported. consider removing it